src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -39,7 +39,7 @@ public static XmlDocument GetXmlDomFromHtml(string content, bool includeXmlDecla
 
 			// It also likes to insert newlines before <b>, <u>, and <i>, and convert any existing whitespace
 			// there to a space.
-			content = new Regex(@"<([ubi])>").Replace(content, "REMOVEWHITESPACE<$1>");
+			content = new Regex(@"<([ubi]|em|strong)>").Replace(content, "REMOVEWHITESPACE<$1>");
```

Comment:
I suppose there's a good reason why this one doesn't have 'span' like the others but I wish one of us...probably me...had commented.